packages/interaction-kit/src/components/button.ts

```diff
@@ -142,28 +145,40 @@ export class Button
 	extends ButtonBase<Exclude<ButtonStyle, ButtonStyle.Link>>
 	implements SerializableComponent, Executable<ButtonInteraction>
 {
-	#customID: ButtonArgs["customID"];
+	#customId: ButtonArgs["customId"];
 	onInteraction: ButtonArgs["onInteraction"];
-
-	constructor(options: ButtonArgs) {
+	trigger?: ButtonArgs["trigger"];
```

Comment:
```suggestion
	trigger: ButtonArgs["trigger"];
```
I think optional isn't necessary here, right? Since `ButtonArgs["trigger"]` is already optional and `Select` don't use it as optional.

Reply:
good call

Reply:
Looks like maaaaybe this makes TS hinting harder to read - need to double check before merging